hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+  > Important: The Microservice hosting feature must be activated on your tenant, otherwise your request will return an error message like “security/Forbidden, access is denied”. This feature is not assigned to tenants by default, so trial accounts won’t have it. Contact [product support](https://cumulocity.com/guides/welcome/contacting-support/) so that we can assist you with the activation. Note that this is a paid feature. Microservice SDK for Java - Cumulocity IoT Guides
+- contents of this repository
+
+### Instructions
+1. Creating a test user
+    - From the Application Switcher select Administration
+    - In the Navigation pane expand Accounts and go to Users
+    - Click on 'Add user', fill in the form for the new user, untick "Send password reset link as email" and provide a password
+    - Click Save
+    - Enable notification support:
+        - Using Navigation pane, go to Accounts and then Roles, click on "Add global role"
+        - Name the role "Notifications" and from the list of permissions tick "Notification 2" in Admin column
+        - click Save
+        - go to Accounts then Users
+        - in the list, find the username created a few steps before and under "Global roles" open the dropdown
+        - find and tick "Notifications" and click "Apply"
+2. Creating a test device
+    - open the demo directory
+    - edit the `device.py` script and provide the following credentials for your newly created user as well as the platform url:
+      ```python
+      _tenant_id = ''
+      _username = ''
+      _password = ''
+      _platform_url = ''
+        ```
+    - create a device using `python3 device.py device` and make note of its id, example:
+       ```console
+       user@host:~$ python3 device.py device
+       user@host:~$ Device id: 123
+       ```
+3. Build the microservice
+    - edit the `application.properties` located in `src/main/resources`
+        - supply the device id in `example.source.id`
```

Comment:
actually say which patter to replace "<DEVICE_ID>".

Reply:
sorry, what did you mean Andre?

Reply:
supply the device id in 'example.source.id' ---> replace "<DEVICE_ID>" in .... with the device id.